web/packages/shared/components/UnifiedResources/FilterPanel.tsx

```diff
@@ -568,16 +543,17 @@ const IncludedResourcesSelector = ({
           size="small"
           onClick={handleOpen}
         >
-          Availability
+          Access Requests
           <ChevronDown ml={2} size="small" color="text.slightlyMuted" />
-          {availabilityFilter.canRequestAll === true &&
-            availabilityFilter.mode !== 'none' && <FiltersExistIndicator />}
+          {availabilityFilter.mode === 'accessible' && (
```

Comment:
I think it doesn't work correctly for `show_resources: 'accessible'`. It shows the indicator when the toggle is not checked (so it's in a default state):
<img width="432" alt="image" src="https://github.com/user-attachments/assets/e7d6a850-fefe-4ca0-90f4-c6de990a7907">

Reply:
The default state is "on" so the indicator _should_ show when `accessible` is the value